packages/shared/src/lib/image.ts

```diff
@@ -11,8 +11,6 @@ export const cloudinary = {
       'https://daily-now-res.cloudinary.com/image/upload/v1665224467/public/recommended.png',
     yourFeed:
       'https://daily-now-res.cloudinary.com/image/upload/v1667998924/public/your_feed.png',
-    supercharge:
-      'https://daily-now-res.cloudinary.com/image/upload/v1665563545/public/supercharge.png',
```

Comment:
I think it might be best to load the SVG from cloudinary as well, instead of in the files.
CC: @sshanzel / @idoshamun what do you guys think?

Reply:
The good thing about cloudinary is the image optimizations which are not applied on svg. So for me it doesn't matter. Your call :)

Reply:
I think generally bundling svg with code is better. Since we lazy load by default it is even better for that case.